server/builtin/app.go

```diff
@@ -149,7 +156,7 @@ func App(conf config.Config) apps.App {
 	}
 }
 
-func (a *builtinApp) Roundtrip(_ apps.App, creq apps.CallRequest, async bool) (out io.ReadCloser, err error) {
+func (a *builtinApp) Roundtrip(ctx context.Context, _ apps.App, creq apps.CallRequest, async bool) (out io.ReadCloser, err error) {
```

Comment:
I thought about `Roundtrip` using `request.Context` instead of `context.Context` would would allow the built-in app to re-use the  `request.Context` from the `Proxy`, but that would make the other uses of `Roundtrip` a bit weird.